docs/getting_support.rst

```diff
@@ -3,45 +3,435 @@
+.. |2FA backup codes| image:: images/support/2fa_backup_codes.png
+
+Other account settings
+----------------------
+In addition to two-factor authentication, you can also configure your local
+time zone in the account settings.
+
+We encourage you to leave the notification setting as the default:
+"For any event on all my projects".
+
+|Account settings|
+
+.. |Account settings| image:: images/support/account_settings.png
+
+.. _encrypted_email_overview:
+
+Encrypted Email
+---------------
+
+One of Redmine's strengths as a ticketing system is its powerful support
+for email-based workflow. You can use email to create new issues, reply
+to existing issues, and be notified of updates to issues that are
+relevant to you.
+
+While many people find email-based workflows convenient, email is
+unfortunately insecure by default. Freedom of the Press Foundation takes
+the security of every SecureDrop instance seriously; therefore, we
+require the use of encryption for support requests because they may
+contain sensitive information about your SecureDrop instance.
+
+The web interface workflow is automatically encrypted thanks to HTTPS.
+Supporting a secure email-based workflow is more difficult because email
+is unencrypted by default. Our solution is to combine Redmine's
+excellent email-based workflow with OpenPGP encryption, which we already
+use to communicate with many SecureDrop administrators and journalists.
+
+What if I don't want to use encrypted email?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+That's fine! You can do everything through the web interface that you
+can do through email.
+
+If you don't do the setup process for receiving encrypted emails from
+our support server, you will still receive email alerts for changes to
+issues in your project, but the content of the email will not be
+included. This is called a "filtered" email.
+
+|FilteredEmail|
+
+We encourage you to use these filtered emails as a reminder to login to
+your Redmine account and check the content of the corresponding updates
+to an issue through the web interface. We like to think of them as "poor
+man's push notifications".
+
+.. note:: We welcome feedback on how we could make notifications for
+          this Redmine system more convenient for you.
+
+.. |FilteredEmail| image:: images/support/filtered_email.png
+
+Setting up Encrypted Email
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to use this encrypted email workflow, you need to:
+
+1. Provide **your public key** to the support server
+2. Import the **server's public key** into your local GPG keyring.
+
+The following documentation explains how to perform these steps to get
+the encrypted email workflow working. This documentation assumes you are
+familiar with PGP/GPG. We encourage you to use your preferred GPG key
+management tool and email client.
+
+If you do not know how to do things like import and export GPG keys,
+the encrypted email workflow is not for you. We recommend you stick to
```

Comment:
The advice makes sense (don't bother unless you already know PGP), but as written now it feels unnecessarily personal?

"If you are not already familiar with using PGP/GPG keys,we would recommend using the web-based workflow instead of setting up the encrypted email workflow." or something.